core/templates/dev/head/pages/skill_editor/questions_tab/questions_tab_directive.html

```diff
@@ -0,0 +1,109 @@
+<md-card class="oppia-page-card oppia-long-text" ng-if="!questionEditorIsShown">
+  <div class="questions">
+    <h3 class="form-heading"> Questions included in this topic </h3>
```

Comment:
This text is no longer accurate in the case of viewing from the skill editor.

Reply:
Done.